ui/app/models/auth-method.js

```diff
@@ -94,7 +104,7 @@ export default class AuthMethodModel extends Model {
       'accessor',
       'local',
       'sealWrap',
-      'config.{listingVisibility,defaultLeaseTtl,maxLeaseTtl,tokenType,auditNonHmacRequestKeys,auditNonHmacResponseKeys,passthroughRequestHeaders}',
+      'config.{listingVisibility,defaultLeaseTtl,maxLeaseTtl,tokenType,auditNonHmacRequestKeys,auditNonHmacResponseKeys,passthroughRequestHeaders,allowedResponseHeaders,pluginVersion}',
```

Comment:
didn't add any `user_lockout_config` here because they are only listed under `tune` attrs in the [docs](https://developer.hashicorp.com/vault/api-docs/system/auth#tune-auth-method)